### DISCUSSION TITLE
Bridged AP mode

### DISCUSSION THREAD

**Discussion by lifros** [RaspAP/raspap-webgui#811] [General]
Hello,
in Bridge AP mode the system will automatically set the following options in dhcpcd.conf:
- interface br0
- denyinterfaces wlan0 eth0

But this not working because eth0 get an IP address from the DHCP server before the deny policy was applied.
The correct sequence is as follow:
- denyinterfaces wlan0 eth0
- interface br0

Thanks and best regards.
Luca Di Natale

**Comment by billz** [comment]
I moved this from issues as it was missing the required template, but feel it's a valid discussion topic.

Bridged AP mode is entirely thanks to @Taikuh's contribution in #523. Relevant discussion [here](https://github.com/raspberrypi/documentation/issues/1423#issuecomment-596086168) between @Taikuh and @epoch1970, who authored the official RPi bridged AP documentation:

> So 'interface br0' at the top of the file relies only on dhcpcd's built-in defaults and the lease would not reflect options added by the user. Not good.

> It is just a bit awkward to say "add deyninterface to the beginning of the file" and "add interface br0 at the end of the file"... That's why I bunched the 2 together.

The official docs were indeed later updated to include `interface br0` at the end of dhcpcd.conf.

It's not entirely clear how dhcpcd treats the ordering of these options. But if the above solves issues for others I'm all for updating this project accordingly.

**Reply by billz** [reply]
Pushed 871d535
